mis_builder/models/mis_report_instance.py

```diff
@@ -431,6 +430,30 @@ def _default_company_id(self):
     date_from = fields.Date(string="From")
     date_to = fields.Date(string="To")
     temporary = fields.Boolean(default=False)
+    mis_report_view = fields.Text(string='MIS Report view',
+                                  compute='_compute_mis_report_view')
+
+    def _compute_mis_report_view(self):
+        """This method is dummy because we use this field as a placeholder
+        to render the actual mis report instance in the html format using the
+        widget"""
+        for rec in self:
+            rec.mis_report_view = ""
```

Comment:
You need to use a dummy field in order to be able to render the form view into this field.

 I tested the various methods to render the dynamic view. The method used in v10, using the declaration of <widget>, https://github.com/OCA/mis-builder/blob/10.0/mis_builder/views/mis_report_instance.xml#L17, does not work in v11 anymore.
I tried using client actions as used in https://github.com/odoo/odoo/blob/11.0/addons/stock/views/stock_picking_views.xml#L235, but this would not let me  add the view to the dashboard.

Perhaps there's another way, but I could not find it.

Reply:
Perhaps we could bind the widget to the `id` field? So we could use the same widget on any m2o field that points to a `mis.report.instance`?

Reply:
The problem is that with this approach we need to define  <field name="mis_report_view" widget="mis_builder_widget">. And it is "mis_report_view" the field that renders the full representation. I would not know how to bind the widget to the id field.

Reply:
But where is that `mis_report_view` field computed with a real value? I can't find it in the code.

Reply:
The content is defined in the _render method in the JS side https://github.com/OCA/mis-builder/blob/11.0-dev/mis_builder/static/src/js/mis_builder_widget.js#L44

Reply:
So, the field has a widget, and the widget renders the content of this field. It really does happen to need any content from the field itself.

Reply:
Yes but that does not use the `mis_report_view` field at all. So the widget could be bound to any field and the result would be identical?

Reply:
Do you suggest <field name=id widget="mis_builder_widget">?

Reply:
https://github.com/OCA/mis-builder/pull/67/commits/fd26f817ad5582028186b983368299cb31e021db is a first simplification.